src/mds/MDCache.cc

```diff
@@ -11997,6 +12001,8 @@ void MDCache::repair_dirfrag_stats_work(MDRequestRef& mdr)
   if (!good_fragstat) {
     if (pf->fragstat.mtime > frag_info.mtime)
       frag_info.mtime = pf->fragstat.mtime;
+    if (pf->fragstat.change_attr > frag_info.change_attr)
+      frag_info.change_attr = pf->fragstat.change_attr;
```

Comment:
I'm having trouble reasoning about this right now. When can a client see an individual frag's change_attr? are there any negative consequences if eg it sees the change_attr from a local operation, then a scatter-gather happens and it sees the same directory change_attr despite there being new changes in view from another frag?